### PR TITLE
IA-3

### PR DESCRIPTION
**Pull request by mammerman**
Updates to IA-3(1) - IA-3(4)

### CODE REVIEW

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 1]
```diff
@@ -192,27 +190,24 @@
...
+  implementation_status: partial
   narrative:
     - key: a
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-319
+        The system does not provide addresses to devices.  However, it does utilize a private IP space for 
+        communicable endpoints.  Within this space address lease parameters are irrelevant because the endpoints
+        themselves are ephemeral.  A single endpoint instance cannot obtain multiple addresses.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]: What is a "communicable endpoint"? Reads as though every OpenShift API endpoint could be a container with its own IP address. 

Spirit of text looks good though! Could you copy edit to prepare it to be external-facing?

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 2]
```diff
@@ -192,27 +190,24 @@
...
     - key: b
       text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-319
+        Addresses are tied to deployed communicable endpoints, which are audited.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]: Unsure what this means. Could detail be given around how OpenShift containers are ephemeral, and when provisioned or redeployed, receive a new IP address/lease?

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 3]
```diff
@@ -192,27 +190,24 @@
...
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-320
+        This is an organizational control and is not applicable to this system.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]: What capabilities of OpenShift support this process? 

For example, documentation around how OpenShift deploys an image, verifies it prior to placing into production, assigns an IP address, and then any future changes are done through redeployments (that iterate this process).

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Pull request closed without merging**